packages/views.client.unstable/src/client.ts

```diff
@@ -14,93 +14,106 @@
  * limitations under the License.
  */
 
-import type {
+import {
   HostMessage,
```

Comment:
why isn't this a type any more

Reply:
there's a constant version string im referencing